spiders/cclonline.py

```diff
@@ -9,6 +9,17 @@ class Cclonline_spider(scrapy.Spider):
         "https://www.cclonline.com/",
     ]
 
+    custom_settings = {'RETRY_HTTP_CODES': [403],
```

Comment:
What is this for ?

Reply:
ccloin wala spider pa test kar ra tha. us ka liya ya set kia tha ager 403 ho to retry kara